tests/attr/test_feature_permutation.py

```diff
@@ -99,15 +99,15 @@ def forward_func(x):
             self.assertTrue((attribs[:, 1] != 0).all())
             self.assertTrue((attribs[:, 2] != 0).all())
 
-    def test_multi_input(self):
+    def test_multi_input(self) -> None:
         batch_size = 20
         inp1_size = (5, 2)
         inp2_size = (5, 3)
 
         labels = torch.randn(batch_size)
 
-        def forward_func(*x):
-            y = 0
+        def forward_func(*x: Tensor) -> Tensor:
+            y = torch.zeros(x[0].shape[0:2])
```

Comment:
nit: It might be more concise to just have torch.zeros_like(x[0]) here.

Reply:
Replace it with ```torch.zeros_like(x[0])``` will give the following Runtime Error in testing:
```
RuntimeError: The size of tensor a (2) must match the size of tensor b (5) at non-singleton dimension 2
```